test/functional/feature_init.py

```diff
@@ -272,7 +271,14 @@ def break_wait_test(self):
             # returns early it will return the current block height.
             self.log.debug(f"Calling waitforblockheight with {self.rpc_timeout} sec RPC timeout")
```

Comment:
nit: I think the comment should say "with 2x RPC timeout (2x {self.rpc_timeout} sec)"

Reply:
No? The rpc timeout is half in reality, since it is never re-set after the init loop, which sets it to half.

edit: Also, this seems unrelated to the changes here. I am not touching this line.